specifyweb/frontend/js_src/lib/localization/forms.ts

```diff
@@ -986,6 +986,12 @@ export const formsText = createDictionary({
     'uk-ua': 'Використовуйте локалізовані назви полів',
     'de-ch': 'Verwenden Sie lokalisierte Feldbezeichnungen',
   },
+  showLocalizedLabels: {
```

Comment:
(optional) to better match the naming of the `useFieldLabels` option, could rename this to `showFieldLabels` and `showDataModelLabels`